app/javascript/components/forewords/Foreword.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div
+    class="foreword"
+    :class="{ active: isActive }"
+  >
+    <div
```

Comment:
Change made, this is a nice improvement

Reply:
I've changed that div to be a button with type="button" and I've added styles to prevent it assuming the default button border/background. I haven't stopped it from having a dotted outline when focused, which I figure is important for accessibility. Looks okay I think.

<img width="330" alt="image" src="https://user-images.githubusercontent.com/56026099/120621347-b3a2fe80-c455-11eb-92d1-76bc1168da8e.png">
<img width="581" alt="image" src="https://user-images.githubusercontent.com/56026099/120621624-f369e600-c455-11eb-94b0-fb5d873f54ed.png">

Reply:
Wunderbar - danke! Yep, leaving the outline is a good call 👍